test/integration/node-specific/bson-options/utf8_validation.test.ts

```diff
@@ -176,7 +93,9 @@ describe('utf8 validation with cursors', function () {
       if (providedBuffer.includes(targetBytes)) {
         if (providedBuffer.split(targetBytes).length !== 2) {
           sinon.restore();
-          const message = `too many target bytes sequences: received ${providedBuffer.split(targetBytes).length}\n.  command: ${inspect(deserialize(args[0]), { depth: Infinity })}`;
+          const message = `too many target bytes sequences: received ${
+            providedBuffer.split(targetBytes).length
+          }`;
           throw new Error(message);
```

Comment:
We removed this in the other PR because this will just crash before generating the error IIRC

Reply:
We removed deserialzation of the OP_MSG.  We left this in the 6.8 PR.

Reply:
Ah my local copy was out of date. whoops!